src/js/medium-editor.js

```diff
@@ -1029,6 +1031,64 @@ if (typeof module === 'object') {
             return this;
         },
 
+
+        bindDragDrop: function () {
+            var self = this,
+                i;
+
+            if (!self.options.imageDragging) {
+                return;
+            }
+
+            var className = 'medium-editor-dragover';
+            var onDrag = function (e) {
+                e.preventDefault();
+                e.dataTransfer.dropEffect = "copy";
+
+                if (e.type === "dragover") {
+                    this.classList.add(className);
+                } else {
+                    this.classList.remove(className);
+                }
+            };
+
+            var onDrop = function (e) {
+                var files;
+                e.preventDefault();
+                e.stopPropagation();
+                files = Array.prototype.slice.call(e.dataTransfer.files, 0);
+                files.some(function (file) {
+                    if (file.type.match("image")) {
+                        var fileReader = new FileReader();
+                        fileReader.readAsDataURL(file);
+
+                        var id = 'medium-img-' + Math.random();
```

Comment:
If you drag and drop two files at a time, Date.getTime() would have the same value